### DISCUSSION TITLE
🔗 Improvements in the loop carousel widget: amount of slides to display, and hide arrows responsively (Duplicate of #24623)

### DISCUSSION THREAD

**Discussion by witty-code** [elementor/elementor#22631] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

In the loop carousel widget, the 'Slides to display' and 'Slides on Scroll' settings are limited to 8
I have a client who wants to display 12 items in widescreens, but the current setup limits it

![image](https://github.com/elementor/elementor/assets/113376656/886ceb7a-caaa-4c55-b306-196c84bc67b5)

In addition, since on wide screens it shows all slides, the result is that the total amount of slides is the same as the amount of slides actually displayed, this causes the arrows (on the right and left side) to be displayed but not active. If he cancels the display of the arrows, this will cause them to be hidden even on narrower screens, where scrolling between the slides is actually consumed

### Describe the solution you'd like

I suggest allowing a numerical value to be set manually in the 'Slides to display' and 'Slides on Scroll' fields
It is also useful to allow the display and hiding of the scroll arrows to be configured responsively for the different breakpoints

**Comment by corgalore** [comment]
I would like to add to this request. Please support decimal values in the Slides to Display control. This will make use of the Swiper native ability to show partial slides. It would be even better if the control was only a default value and a repeater was also available to set the Slides to Display for multiple breakpoints. This is also natively supported in the Swiper and I've implemented it in some custom widgets with success.

The repeater could look something like this:
![image](https://github.com/elementor/elementor/assets/554935/ab07af00-6aec-4f2d-a82c-c25848132c50)

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #24623 

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
